 2013-03-16  Jonathan Wakely  <jwakely.gcc@gmail.com>
 
	PR libstdc++/56468
	* libsupc++/exception_ptr.h (type_info): Declare.

--- libstdc++-v3/libsupc++/exception_ptr.h	(revision 196708)
+++ libstdc++-v3/libsupc++/exception_ptr.h	(revision 196709)
@@ -44,6 +44,8 @@
 
 namespace std 
 {
+  class type_info;
+
   /**
    * @addtogroup exceptions
    * @{
@@ -140,7 +142,7 @@
       friend bool 
       operator==(const exception_ptr&, const exception_ptr&) throw();
 
-      const type_info*
+      const class std::type_info*
       __cxa_exception_type() const throw();
     };
 
